gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/writer/FileAwareInputStreamDataWriterTest.java

```diff
@@ -402,10 +409,169 @@ public void testCommit() throws IOException {
     // previously existing paths should not have permissions changed
     fileStatus = this.fs.getFileStatus(existingOutputPath);
     Assert.assertEquals(fileStatus.getPermission(), existingPathPermission);
-
     Assert.assertFalse(this.fs.exists(writer.stagingDir));
   }
 
+  @Test
+  public void testCommitWithAclPreservationWhenAncestorPathsAbsent() throws IOException {
+    String fileName = "file";
+    // Asemble destination paths
+    String destinationExistingToken = "destination";
+    String destinationAdditionalTokens = "path";
+    Path destination = new Path(new Path(new Path("/", destinationExistingToken), destinationAdditionalTokens), fileName);
+    Path destinationWithoutLeadingSeparator = new Path(new Path(destinationExistingToken, destinationAdditionalTokens), fileName);
```

Comment:
aren't these two equivalent:
```
Path destWoLS = ...
Path destination = new Path("/", destWoLS);
```
?

Reply:
ahh... that's a good catch. I essentially borrowed these paths from old test cases... I have made the changes and pushed a latest commit for the same!